### ISSUE TITLE
Expressing octal numbers with leading 0 deprecated? (in examples)

### ISSUE THREAD

**Issue by OliverColeman** [VeliovGroup/Meteor-Files#531]
After updating to Meteor 1.6 the `permission` and `parentDirPermissions` settings, represented as octal numbers by way of a leading 0, in my Files collections started resulting in the server crashing. I actually wasted a fair bit of time on this because the error report listed the wrong line number and googling the key words in the error report didn't come up with much. It looks like the [(new) standard](https://github.com/lukehoban/es6features#binary-and-octal-literals) is to represent octals like `0o0774` (instead of `0774`). 

I don't know whether vanilla Meteor 1.6 installs will have this problem or whether it's specific to my set-up (my project is using babel with the 'env' preset). But it at least might be worth mentioning this in the Wiki code if not updating it. I also just wanted to have a record of this somewhere where it might be found by others googling in vain!

**Comment by dr-dimitru** [comment]
@OliverColeman thank you for pointing onto this.
Just to be clear for other folks, who may run into the same issue:
```js
// These are all equal:
420;
parseInt('755', 8);
0o755;
0o0755;
0755;
```

**Comment by dr-dimitru** [comment]
Thank you, docs and wiki are updated. Published [`v1.9.3`](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.9.3)
